docs/en/docs/help-fastapi.md

```diff
@@ -91,6 +132,57 @@ You can <a href="https://github.com/tiangolo/fastapi/issues/new/choose" class="e
 
 **Note**: if you create an issue, then I'm going to ask you to also help others. 😉
 
+## Review Pull Requests
+
+You can help me review pull requests from others.
+
+Again, please try as best to be kind. 🤗
+
+---
+
+Here's what to have in mind and how to review a pull request:
+
+### Understand the problem
+
+* First, make sure you **understand the problem** that the pull request is trying to solve. It might have a longer discussion in an issue.
```

Comment:
```suggestion
* First, make sure you **understand the problem** that the pull request is trying to solve. It might have a detailed discussion in an issue.
```